ducktape/tests/loader.py

```diff
@@ -29,6 +31,28 @@ class TestLoader(object):
     def __init__(self, session_context):
         self.logger = session_context.logger
 
+    def parse_discovery_symbol(self, discovery_symbol):
```

Comment:
Update unit tests for the new syntax? I think this should just be a couple of lines to add a new test method to the existing tests.